### ISSUE TITLE
[Machine Learning] Implementing Bagging Algorithms in Python

### ISSUE THREAD

**Issue by josephgatura328** [section-engineering-education/engineering-education#6189]
#### Proposed title of article
[Machine Learning] Implementing Bagging Algorithms in Python

#### Proposed article introduction
Bagging, also known as bootstrap aggregation, is the ensemble learning method that is commonly used to reduce variance within a noisy dataset. In bagging, a random sample of data in a training set is selected with replacement, this means that the individual data points can be chosen more than once.

After several data samples are generated, these weak models are then trained independently. Depending on the type of task (regression or classification) the average or majority of those predictions yield a more accurate estimate.  Bagging algorithms create several subsets of data from training samples chosen randomly with replacement. Each collection of subset data is used to train their decision trees.

In this tutorial, we will use the `BaggingClassifier` algorithm to train our model. We will build a diabetes prediction model. `BaggingClassifier` algorithm will fine-tune the model until we have a model with the best accuracy score.

#### Key takeaways 
1. Data preprocessing.
2. What is a bagging algorithm?
3. Initializing the `BaggingClassifier` algorithm.
4. Using the `BaggingClassifier` algorithm to build a diabetes prediction model.
5. Comparing the results when using an individual algorithm such as Decision Tree and Random Forest algorithms.

#### Article quality
This tutorial is unique because we will use two approaches in building the model. The first approach is to use an individual algorithm such as the Decision tree algorithm or the Random Forest algorithm in building the model. The second approach is to build the same model using the  `BaggingClassifier` algorithm. We then compare the accuracy scores of the two approaches. This will enable us to see how the Bagging algorithm improves the model's accuracy score. 

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by WanjaMIKE** [comment]
Seems like a helpful topic. Let's ensure that it adds value to the community.  It should be different from existing illustrations such as https://insidelearningmachines.com/build-a-bagging-classifier-in-python/ Thanks
